Makefile

```
@@ -2,44 +2,29 @@ help:           ## Show this help.
 	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
 
 clean-start:       ## Clean, setup then start.
-clean-start: stop-all all
+clean-start: stop-all up-all
 
 stop-all:
 	docker stop $$(docker ps -q)
 
-all:
-	docker-compose \
-		-f docker-compose.yml -f docker-compose-demoui.yml -f docker-compose-keyshare.yml \
-		up --remove-orphans -d
+up-all:
+	docker-compose -f docker-compose.yml up --remove-orphans -d
 
```

It would be good to have one with build?
```suggestion
up-all-build:
         docker-compose -f docker-compose.yml up --remove-orphans -d --build
```

---

Yeah, would be!